test/format/format.sh

```diff
@@ -437,13 +442,22 @@ format()
 	fi
 
 	cmd="$format_binary -c "$config" -h "$dir" -1 $args quiet=1"
-	verbose "$name: $cmd"
+	echo "$name: $cmd"
```

Comment:
Feel the command dump is useful for failure troubleshooting, so turn it on by default.